example/expression/solve_condition_stp.py

```diff
@@ -44,7 +43,7 @@ def emul_symb(ir_arch, mdis, states_todo, states_done):
 
         print 'Run block:'
         print irblock
-        addr = symbexec.eval_updt_irblock(irblock)
+        addr = symbexec.eval_updt_irblock(irblock, step=True)
```

Comment:
:warning: the `step=True` wasn't there in the previous version. It is on purpose?

Reply:
Correct!
done